questions/includes/fedora/coding/python.yml

```diff
@@ -14,7 +14,7 @@ tree:
 
     - title: Firewalld
       subtitle: a dynamically managed firewall with support for network zones
-      href: http://www.firewalld.org/contribute/
```

Comment:
oh weird, maybe it's just where i am at currently (Czech Republic). i'm totally getting 404s on that page.